packages/camera_driver/src/camera_node.py

```diff
@@ -8,15 +8,19 @@
 from threading import Thread
 
 # for Jetson Nano
-import cv2
-import atexit
-import threading
-from cv_bridge import CvBridge, CvBridgeError
+if (os.environ.get('ROBOT_HARDWARE') == "jetson_nano"):
```

Comment:
@carmen-sc I introduced this utility function, always prefer Enums to strings
https://github.com/duckietown/dt-commons/blob/1361dc31a0cb7531016ded9de499fc02032499f8/packages/dt_device_utils/device.py#L38